examples/basic/eventcatalog.config.js

```diff
@@ -8,6 +8,9 @@ module.exports = {
     alt: 'EventCatalog Logo',
     src: 'logo.svg',
   },
+  analytics: {
+    googleAnalyticsTrackingId: 'G-Y0XQQQQQQQ',
```

Comment:
Think we can remove this from the example config file 👍

We use this config file to develop against locally, if this is real (or not) think it will generate noise for us locally. Think the docs should help people learn how to use/implement it :)

Reply:
I'll remove it